GVRf/Framework/framework/src/main/jni/engine/renderer/vulkan_renderer.h

```diff
@@ -119,6 +112,9 @@ class VulkanRenderer: public Renderer {
                                                const TextureParameters* texture_parameters, int number_views);
     virtual RenderTexture* createRenderTexture(int width, int height, int sample_count, int layers) { }
     virtual RenderTexture* createRenderTexture(const RenderTextureInfo&);
+    RenderTexture* createMonoscopicRenderTexture(int width, int height, int sample_count,
```

Comment:
createRenderTexture should be enough. Can't have monoscopic-specifics in this interface.